onyo/main.py

```diff
@@ -245,7 +256,7 @@ def setup_parser() -> argparse.ArgumentParser:
         'unset',
         description=commands.unset.__doc__,
         formatter_class=SubcommandHelpFormatter,
-        help=commands.unset.__doc__
+        help='Remove keys for matching assets.'
```

Comment:
Same as for `onyo set` -- there is no more matching for unset than there is for `onyo rm`.

Reply:
Fixed and pushed.